seednode/install_seednode_debian.sh

```diff
@@ -69,6 +69,10 @@ sudo -H -i -u "${ROOT_USER}" git config --global advice.detachedHead false
 sudo -H -i -u "${ROOT_USER}" git clone --branch "${BISQ_REPO_TAG}" "${BISQ_REPO_URL}" "${ROOT_HOME}/${BISQ_REPO_NAME}"
 
 echo "[*] Installing Tor"
+sudo -H -i -u "${ROOT_USER}" wget -qO- https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gp
+g --dearmor | tee /usr/share/keyrings/tor-archive-keyring.gpg >/dev/null
```

Comment:
It seems this command is split over two lines.

Reply:
☝️ @Emzy Could you please comment on this? Thanks!

Reply:
Good catch. I fixed it.